### PR TITLE
Cloud: update key value list for web locations

### PR DESCRIPTION
**Pull request by hguthrie**
## Purpose of this pull request

This pull request (PR) is to update the web key value list in the Cloud topic for `.magento.app.yaml` properties. This also addresses a request in GH issue #8021 . 

## Affected DevDocs pages

https://devdocs.magento.com/cloud/project/magento-app-properties.html#web

whatsnew
Updated the [`web` property used in the `magento.app.yaml` properties topic](https://devdocs.magento.com/cloud/project/magento-app-properties.html#web) in the Cloud guide.

### CODE REVIEW

**Comment by hguthrie** [comment]
Hi @NadiyaS  could someone from Cloud team take a look at this list for the web properties for `.magento.app.yaml`?
It was out of date and was missing some key values that we include in our examples. I reordered the list so they are alphabetical. And I added a couple that were missing, but need confirmation and help defining defaults or purpose.

File: src/cloud/project/magento-app-properties.md [hunk 1]
```diff
@@ -73,20 +73,26 @@ See [Services]({{ site.baseurl }}/cloud/project/services.html) for a full list o
...
+`index` | List the static files to serve your application, such as the `index.html` file. This key expects a collection. This only works if access to the file or files is "allowed" by the `allow` or `rules` key for this location.
+`rules` | Specify overrides for a location. Use a regular expression to match a request. If an incoming request matches the rule, then regular handling of the request is overridden by the keys used in the rule.
+`passthru` | Set the URL used in the event that a static file or PHP file cannot be found. Typically, this URL is the front controller for your applications, such as `/index.php` or `/app.php`.
+`root` | Set the path relative to the root of the application that is exposed on the web. Typical values include `/public` and `/web`.
```
Comment (NadiyaS) [inline src/cloud/project/magento-app-properties.md:94]: > Typical values include `/public` and `/web`.
I would remove this as this is not the case for Cloud

**Review by NadiyaS — APPROVED** [review]

File: src/cloud/project/magento-app-properties.md (hunk 1 above)
Comment (hguthrie) [reply src/cloud/project/magento-app-properties.md:94]: Changing to pub. Thanks!

**Comment by hguthrie** [comment]
running tests

**Pull request merged**